R/pinp.R

```diff
@@ -11,6 +11,51 @@
 #' @return R Markdown output format to pass to
 #' \code{\link[rmarkdown:render]{render}}
 #'
+#' @section Document options:
+#' Various aspects of pdf document can be customized by setting either standard \pkg{rmarkdown}
+#' options, or any of the following options (which are shown in alphabetical order) in the 
+#' document metadata:
+#'
+#' \describe{
```

Comment:
Have you tried `roxygen2` with Markdown enabled yet? 

https://cran.r-project.org/web/packages/roxygen2/vignettes/markdown.html#lists

Support exists for both `enumerate` and `itemize` lists. I manually recreate the `describe` list format by modifying the `itemize` list to mimic it using embedded segments. e.g.

```r
#' - `code`
#'     - description
```

Reply:
I happen to dislike what it requires in DESCRIPTION.

Reply:
Aye, 'tis not a pretty sight; but, we can manually enable it on a per-chunk basis via:

```r
#' @md
```

Reply:
With that being said, I'm fine with using the traditional latex markup.